apps/codeforafrica/src/lib/api.ghost.js

```diff
@@ -0,0 +1,88 @@
+import GhostContentAPI from "@tryghost/content-api";
+
+import convertToCamelCase from "@/codeforafrica/utils/camelcaseKeys";
+
+const { GHOST_URL } = process.env;
+// const { GHOST_URL } = "https://longform.codeforafrica.org";
+const { GHOST_API_KEY } = process.env;
```

Comment:
1. Don't destructure environment variables. See a note about this in the [docs](https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables).
2. Include `GHOST_API_VERSION` as part of envs as well (defaulting to `"v5.0"`);